### PR TITLE
Debug addressbookjson loading

### PR DESCRIPTION
**Pull request by jowhee3011**
Changed saving date and venue

Changed reading of date and venue from address book

### CODE REVIEW

**Review by VedJoshi — COMMENTED** [review]
LGTM

File: src/main/java/seedu/address/storage/JsonAdaptedWedding.java [hunk 1]
```diff
@@ -50,8 +50,8 @@ public JsonAdaptedWedding(Wedding source) {
         } else {
             client = new JsonAdaptedPerson(source.getClient().getPerson());
         }
-        date = source.getDate() == null ? "" : source.getDate().toString();
-        venue = source.getVenue() == null ? "" : source.getVenue().toString();
+        date = source.getDate() == null ? null : source.getDate().toString();
```
Comment (Droas590) [inline src/main/java/seedu/address/storage/JsonAdaptedWedding.java:53]: good use of null

File: src/main/java/seedu/address/storage/JsonAdaptedWedding.java [hunk 2]
```diff
@@ -74,21 +74,21 @@ public Wedding toModelType(List<Wedding> weddingList) throws IllegalValueExcepti
...
         Client finalClient = new Client(client.toModelType(weddingList));
 
-        if (date == null) {
-            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Date.class.getSimpleName()));
+        Date modelDate = null;
+        if (date != null) {
+            if (!Date.isValidDate(date)) {
+                throw new IllegalValueException(Date.MESSAGE_CONSTRAINTS);
+            }
+            modelDate = new Date(date);
```
Comment (Droas590) [inline src/main/java/seedu/address/storage/JsonAdaptedWedding.java:82]: good chunk of codes

**Review by Droas590 — APPROVED** [review]
LGTM

**Pull request merged**